data.json

```
@@ -880,5 +880,11 @@
     "githubUsername": "wordssaysalot",
     "imageUrl": "https://user-images.githubusercontent.com/56651899/118150685-71773600-b430-11eb-95f7-38a7479c5e34.png",
     "issueNumber": 498
+  },
+  {
+    "name": "Carlos Souza",
+    "githubUsername": "CarlosViniMSouza",
+    "imageUrl": "https://raw.githubusercontent.com/CarlosViniMSouza/CarlosViniMSouza/main/images/readme_branco.png",
```

```suggestion
    "imageUrl": "https://raw.githubusercontent.com/CarlosViniMSouza/CarlosViniMSouza/main/images/readme_branco.png",
```
Please don't use the picture uploaded in your Readme here. Just copy paste the raw image in the issue. This will create a github image link and use this link for adding your picture here. Hope this was understandable 😅

---

No, i don't understand, sorry! 😅